homeassistant/components/proxmoxve/config_flow.py

```diff
@@ -0,0 +1,830 @@
+                                },
+                            )
+                            continue
+
+                        if await self._async_endpoint_exists(
+                            f"{self._config[CONF_HOST]}/{self._config[CONF_PORT]}/{node[CONF_NODE]}"
+                        ):
+                            LOGGER.warning(
+                                "The node %s of instance %s:%s already configured, you can remove it from configuration.yaml if you still have it",
+                                node[CONF_NODE],
+                                self._config[CONF_HOST],
+                                self._config[CONF_PORT],
+                            )
+                            async_create_issue(
+                                async_get_hass(),
+                                DOMAIN,
+                                f"import_already_configured_{DOMAIN}_{host}_{port}_{node[CONF_NODE]}",
+                                breaks_in_ha_version="2022.12.0",
+                                is_fixable=False,
+                                severity=IssueSeverity.WARNING,
+                                translation_key="import_already_configured",
+                                translation_placeholders={
+                                    "integration": INTEGRATION_NAME,
+                                    "platform": DOMAIN,
+                                    "host": host,
+                                    "port": port,
+                                    "node": node[CONF_NODE],
+                                },
+                            )
+                            continue
+
+                        self._config[CONF_NODE] = node[CONF_NODE]
+                        self._config[CONF_QEMU] = []
+                        self._config[CONF_LXC] = []
+                        for vm_id in node[CONF_VMS]:
+                            self._config[CONF_QEMU].append(vm_id)
+                        for container_id in node[CONF_CONTAINERS]:
+                            self._config[CONF_LXC].append(container_id)
+
+                        async_create_issue(
+                            async_get_hass(),
+                            DOMAIN,
+                            f"import_success_{DOMAIN}_{host}_{port}_{node[CONF_NODE]}",
+                            breaks_in_ha_version="2022.12.0",
+                            is_fixable=False,
+                            severity=IssueSeverity.WARNING,
+                            translation_key="import_success",
+                            translation_placeholders={
+                                "integration": "Proxmox VE",
+                                "platform": DOMAIN,
+                                "node": node[CONF_NODE],
+                                "host": host,
+                                "port": port,
+                            },
+                        )
+
+                        return self.async_create_entry(
```

Comment:
Here's a point I can't resolve...

Currently, the YAML configuration allows configuring multiple nodes for the same host.
When importing, I check if the node exists on the host before being imported, but when the first node has a valid configuration, I create the configuration entry and the import from that host is terminated even if there are more nodes to be evaluated/configured.

I tried some ways to get around this, but to work the return must be an `async_create_entry` and that ends the import to the host. @bachya do you have any suggestions for dealing with this?

Reply:
I did something similar (create multiple config entries from old YAML) for AirVisual: https://github.com/home-assistant/core/blob/dev/homeassistant/components/airvisual/__init__.py#L304-L316

Reply:
You shouldn't attempt to create new config flows within an existing flow. Instead, during setup, create multiple tasks that each start a new config flow.0

Reply:
I fixed it.  Thanks.